docs/maui/converters/color-to-byte-alpha-converter.md

```diff
@@ -85,18 +85,18 @@ class ColorToByteAlphaConverterPage : ContentPage
     public ColorToByteAlphaConverterPage()
     {
         Content = new VerticalStackLayout
- 		{
- 			Children =
- 			{
- 				new Label()
- 					.Text("The alpha component is:"),
- 				new Label()
- 					.Bind(
- 						Label.TextProperty,
- 						nameof(ViewModel.MyFavoriteColor),
- 						converter: new ColorToByteAlphaConverter())
- 			}
- 		};
+        {
+            Children =
+            {
+                new Label()
+                    .Text("The alpha component is:"),
+                new Label()
+                    .Bind(
```

Comment:
Let's update this binding to use TypedBindings

Reply:
We can't as we don't have the converter support yet

Reply:
Ah, good catch!